cmd/kubeadm/app/apis/kubeadm/types.go

```diff
@@ -101,4 +101,16 @@ type NodeConfiguration struct {
 	NodeName                 string
 	TLSBootstrapToken        string
 	Token                    string
+
+	// TLSDiscoveryRootCAPubKeys specifies a set of public key pins. The root CA
+	// discovered during TLS bootstrapping must match one of these values.
+	// Specifying an empty set disables root CA pinning. Each hash is a hex encoded
```

Comment:
I'd still love to see us do something like base 36 or base 58 here to shorten this.  But that can come in a later PR.